config/jobs/image-pushing/k8s-staging-cluster-api.yaml

```diff
@@ -8,6 +8,7 @@ postsubmits:
         testgrid-dashboards: sig-cluster-lifecycle-image-pushes, wg-k8s-infra-gcb
       decorate: true
       branches:
+        - ^main$
         - ^master$
```

Comment:
I assume this will be dropped in a later PR?

Reply:
correct (this one should have already been merged, it was forgotten in https://github.com/kubernetes/test-infra/pull/23672). Will remove master everywhere in a follow up PR (steps listed here: https://github.com/kubernetes-sigs/cluster-api/issues/3280#issuecomment-923313590)

Reply:
Yup, I just thought based on the name it's the post-rename PR :)